### ISSUE TITLE
output Nan loss

### ISSUE THREAD

**Issue by ChuangLee** [HobbitLong/SupContrast#100]
The code below output Nan loss.
```
    supconloss = SupConLoss(temperature=0.07).cuda()
    data = torch.randn((48, 128))
    labels = torch.arange(0, 24, 0.5).int()
    loss = supconloss(data, labels)
    print(loss)
```

**Comment by ChuangLee** [comment]
uh huh... may it's because there is not normalization. close
